UPGRADING.md

```diff
@@ -1,110 +1,121 @@
-  as possible.
-
-  ```go
-  anteDecorators := []sdk.AnteDecorator{
-      ante.NewSetUpContextDecorator(),
-      // ...
-      ante.NewUnorderedTxDecorator(unorderedtx.DefaultMaxTimeoutDuration, options.TxManager, options.Environment),
-      // ...
-  }
-
-  return sdk.ChainAnteDecorators(anteDecorators...), nil
-  ```
-
-* If the App has a SnapshotManager defined, you must also register the extension
-  for the TxManager.
-
-  ```go
-  if manager := app.SnapshotManager(); manager != nil {
-      err := manager.RegisterExtensions(unorderedtx.NewSnapshotter(app.UnorderedTxManager))
-      if err != nil {
-          panic(fmt.Errorf("failed to register snapshot extension: %w", err))
-      }
-  }
-  ```
-
-* Create or update the App's `Preblocker()` method to call the unordered tx
-  manager's `OnNewBlock()` method.
-
-  ```go
-  ...
-  app.SetPreblocker(app.PreBlocker)
-  ...
-
-  func (app *SimApp) PreBlocker(ctx sdk.Context, req *abci.RequestFinalizeBlock) (*sdk.ResponsePreBlock, error) {
-      app.UnorderedTxManager.OnNewBlock(ctx.BlockTime())
-      return app.ModuleManager.PreBlock(ctx, req)
-  }
-  ```
-
-* Create or update the App's `Close()` method to close the unordered tx manager.
-  Note, this is critical as it ensures the manager's state is written to file
-  such that when the node restarts, it can recover the state to provide replay
-  protection.
-
-  ```go
-  func (app *App) Close() error {
-      // ...
-
-      // close the unordered tx manager
-      if e := app.UnorderedTxManager.Close(); e != nil {
-          err = errors.Join(err, e)
-      }
-
-      return err
-  }
-  ```
-
-</details>
-
-To submit an unordered transaction, the client must set the `unordered` flag to
-`true` and ensure a reasonable `timeout_height` is set. The `timeout_height` is
-used as a TTL for the transaction and is used to provide replay protection. See
-[ADR-070](https://github.com/cosmos/cosmos-sdk/blob/main/docs/architecture/adr-070-unordered-transactions.md) for more details.
+That's it.
+
+### New Modules
+
+Below are some **optional** new modules you can include in your chain. 
+To see a full example of wiring these modules, please check out the [SimApp](https://github.com/cosmos/cosmos-sdk/blob/release/v0.53.x/simapp/app.go).
+
+#### Epochs
+
+⚠️Adding this module requires a `StoreUpgrade`⚠️
+
+The new, supplemental `x/epochs` module provides Cosmos SDK modules functionality to register and execute custom logic at fixed time-intervals.
+
+Required wiring:
+- Keeper Instantiation
+- StoreKey addition
+- Hooks Registration 
+- App Module Registration
+- entry in SetOrderBeginBlockers
+- entry in SetGenesisModuleOrder
+- entry in SetExportModuleOrder
+
+#### ProtocolPool
+
+⚠️Adding this module requires a `StoreUpgrade`⚠️
+
+The new, supplemental `x/protocolpool` module provides extended functionality for managing and distributing block reward revenue.
+
+Required wiring:
+- Module Account Permissions
+  - protocolpooltypes.ModuleName (nil)
```

Comment:
not really sure how verbose we want to get here. i initially put some code stubs but it felt super overwhelming. not sure if this is supposed to be a super long winded document, or just a quick reference as is changing for chains and new stuff i can add.